test_scripts/Smoke/API/001_SetGlobalProperties_PositiveCase_SUCCESS.lua

```diff
@@ -0,0 +1,136 @@
+---------------------------------------------------------------------------------------------------
+-- User story: Smoke
+-- Use case: SetGlobalProperties
+-- Item: Happy path
+--
+-- Requirement summary:
+-- [SetGlobalProperties] SUCCESS on TTS.SetGlobalProperties and UI.SetGlobalProperties
+--
+-- Description:
+-- In case mobile application sends valid SetGlobalproperties_request with "timeoutPrompt"
+-- and/or "helpPrompt" and at least one other valid parameter, SDL must transfer from mobile
+-- app to HMI the both UI.SetGlobalProperties and TTS.SetGlobalProperties. On getting "SUCCESS"
+-- result code from the both HMI-portions, SDL must transfer "resutCode:SUCCESS", success:"true"
+-- to mobile application.
+
+-- Pre-conditions:
+-- a. HMI and SDL are started
+-- b. appID is registered and on SDL
+-- c. appID is in Background, Full or Limited HMI level
+
+-- Steps:
+-- appID requests SetGlobalproperties with timeoutPrompt, helpPrompt and other valid parameters
+
+-- Expected:
+
+-- SDL validates parameters of the request
+-- SDL checks if UI interface is available on HMI
+-- SDL checks if TTS interface is available on HMI
+-- SDL checks if SetGlobalproperties is allowed by Policies
+-- SDL checks if all parameters is allowed by Policies
+-- SDL transfers the UI part of request with allowed parameters to HMI
+-- SDL transfers the TTS part of request with allowed parameters to HMI
+-- SDL receives UI and TTS part of response from HMI with "SUCCESS" result code
+-- SDL transfers response to mobile app
+---------------------------------------------------------------------------------------------------
+
+--[[ Required Shared libraries ]]
+local runner = require('user_modules/script_runner')
+local commonSmokeApi = require('test_scripts/Smoke/commonSmokeApi')
+
+--[[ Local Variables ]]
+local requestParams = {
+	helpPrompt = {
+		{
+			text = "Help prompt",
+			type = "TEXT"
+		}
+	},
+	timeoutPrompt =	{
+		{
+			text = "Timeout prompt",
+			type = "TEXT"
+		}
+	},
+	vrHelpTitle = "VR help title",
+	vrHelp = {
+		{
+			position = 1,
+			image = {
+				value = "icon.png",
+				imageType = "DYNAMIC"
+			},
+			text = "VR help item"
+		}
+	},
+	menuTitle = "Menu Title",
+	menuIcon = {
+		value = "icon.png",
+		imageType = "DYNAMIC"
+	},
+	keyboardProperties = {
+		keyboardLayout = "QWERTY",
+		keypressMode = "SINGLE_KEYPRESS",
+		limitedCharacterList = {"a"},
+		language = "EN-US",
+		autoCompleteText = "Daemon, Freedom"
+	}
+}
+
+--[[ Local Functions ]]
+local function put_file(fileName, self)
```

Comment:
@AKalinich-Luxoft , suggest to move putFile to common.
Smoke suppose basic checks of all RPCs with all possible and valid parameters, so in other scripts PutFile RPC will be used as precondition before sending tested API.

Reply:
@Itileda fixed in 5320c5539bfe24a781724ca9cf32e4ea14839897